packaging/linux/common/khiops_env/khiops_env.in

```diff
@@ -44,6 +44,7 @@ help() {
     echo "  . set to 'false' to allow local file name selection only with a file selection dialog"
     echo "KHIOPS_MPI_HOST_FILE: provide hostfile to mpi."
     echo
+    echo "KHIOPS_MPI_VERBOSE: true or false (default), print messages from mpi (OpenMPI only)."
```

Comment:
Change to:
`echo "KHIOPS_MPI_VERBOSE: true (default) or false, print messages from mpi (OpenMPI only)."`

Reply:
ok